sci-mathematics/sage/sage-9999.ebuild

```diff
@@ -490,7 +490,8 @@ python_install_all(){
 	pushd bin
 
 	dobin sage-native-execute sage \
-		sage-python sage-version.sh
+		sage-python sage-version.sh \
+		math-readline
```

Comment:
This is the wrong place to put this for sage-9999 (it is ok for the other two). sage-9999 which will become sage-8.0 when it is released support both python 2.7 and 3.6. `math-readline` is a python script so it need to be installed by `python_doscript`. So you should move this from the `python_install_all` section to the `python_install` section, under the heading `COMMAND helper scripts`